<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_QRcode</class>
 <widget class="QWidget" name="Form_QRcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>二维码登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <widget class="QLabel" name="label_QR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>获取二维码中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
